test/test_prototype_datasets_utils.py

```diff
@@ -0,0 +1,46 @@
+import sys
+
+import numpy as np
+import pytest
+import torch
+from datasets_utils import make_fake_flo_file
+from torchvision.datasets._optical_flow import _read_flo as read_flo_ref
+from torchvision.prototype.datasets.utils._internal import read_flo, fromfile
+
+
+@pytest.mark.filterwarnings("error:The given NumPy array is not writeable:UserWarning")
```

Comment:
I think this will filter out the warnings so that they don't appear when we run the tests, right?

My suggestion earlier was to make sure that the call to `fromfile` does *not* raise such a warning.

Reply:
> I think this will filter out the warnings so that they don't appear when we run the tests, right?

Nope, that turns `UserWarning`'s that start with the text `"The given NumPy array is not writeable"` into errors. To silence them, the action would have to be `ignore` instead of `error`. See https://docs.python.org/3/library/warnings.html?highlight=warnings#the-warnings-filter for details.